src/app/public/shared/card/card.component.css

```diff
@@ -0,0 +1,31 @@
+.tableRepository__description {
+  width: 300vw;
+  padding: 10px 10px 0px 10px;
+}
+.tableRepository__description--name {
```

Comment:
this should be `tableRepository__name`
give `--modifier` should be a good practice if you have something like:
`container__title--primary`
`container__title--secondary`